data/sql/updates/pending_db_world/rev_1698326815078104700.sql

```diff
@@ -0,0 +1,4 @@
+-- Fire Nova Totem VII - Cast Fire Nova (Rank 7)
+DELETE FROM `creature_template_spell` WHERE (`CreatureID` = 15483);
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
+(15483, 0, 25537, 42328);
```

Comment:
don't be lazy, use UPDATE instead

Reply:
Using update is lazier.

Reply:
the entry already exist, update is the correct way to tackle this.

Reply:
sigh